docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -32,6 +32,16 @@ Excel.run(function (context) {
 });
 ```
 
+### Run options
+
+**Excel.run** has an overload that takes in a [RunOptions](https://docs.microsoft.com/javascript/api/excel/excel.runoptions) object. This contains a set of properties affecting platform behavior during the function's execution. The following are some of the most commonly used option properties:
```

Comment:
"This contains a set of properties affecting platform behavior during the function's execution." -> "This object contains a set of properties that impact platform behavior when the function runs."

FYI, style guide advises to never use the word "execute" or "execution" (presumably since it can be erroneously translated to "kill" or "killing" in other languages) -- always opt for the word "run" (or some variation) instead.